src/commands/common.ts

```diff
@@ -74,3 +78,46 @@ export function parseSize(size: string): number {
   if (isNaN(fileSize)) throw new Error(`Failed to parse: ${size} as a file size`);
   return Math.round(fileSize);
 }
+
+/** Limit fetches to 25 concurrently **/
+const TiffQueue = pLimit(25);
+/**
+ * There is a minor difference between @chunkd/core and @cogeotiff/core
+ * because @chunkd/core is a major version behind, when it upgrades this can be removed
+ *
+ * Because the major version upgrade for chunkd is a lot of work skip it for now (2023-11)
+ *
+ * @param loc
+ * @returns
+ */
+export function createTiff(loc: string): Promise<CogTiff> {
+  const source = fsa.source(loc);
+
+  const tiff = new CogTiff({
+    url: tryParseUrl(loc),
+    fetch: (offset, length): Promise<ArrayBuffer> => {
+      // Limit fetches to 25 concurrently
```

Comment:
This comment looks like it doesn't belong here anymore.

Reply:
Generally you wouldnt need to overwrite the fetch, but we dont currently limit our s3 connections, so the comment is more around why the fetch wrapper is needed. could remove the 25 part as its not defined here.